### PR TITLE
Added a More Common Example to the Wiki

### PR DESCRIPTION
**Pull request by HazyArc14**
Added a more common example to the installation page on the wiki.

## Description
Added an example of how to launch the server with some of the common configs that are used whenever running the server.

## Motivation and Context
This is to show the users what a more common way to run the server is rather than just a basic example and then have to piece everything else together.

## How Has This Been Tested?
It's a wiki page.

## Checklist:
- [x ] My code follows the code style of this project.
- [x ] My change requires a change to the documentation.
- [x ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by HazyArc14** [comment]
Any suggestions for common commands to add?

File: docs/basic-install/index.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
-Basic Installation
-##################
+Installation
+############
 
 These instructions cover an instation from a **release** as well as from a git clone.
```
Comment (sebastienvercammen) [inline docs/basic-install/index.rst]: `instation`

File: docs/basic-install/index.rst [hunk 2]
```diff
@@ -123,10 +123,38 @@ The most basic config you could use would look something like this:
...
   python ./runserver.py -a ptc -u "USERNAME_HERE" -p "PASSWORD_HERE" \
-   -l "a street address or lat/lng coords here" -st 3 -k "maps key here"
+   -l "a street address or lat/lng coords here" -st 3 -k "MAPS_KEY_HERE"
 
 Open your browser to http://localhost:5000 and your pokemon will begin to show up! Happy hunting!
 
+Common Launch Example
+*********************
+
+A more common server config will include multiple accounts, a hash key, and some form of capchta solving.
```
Comment (sebastienvercammen) [inline docs/basic-install/index.rst]: `capchta`

**Comment by HazyArc14** [comment]
@sebastienvercammen Is this what you had in mind? Should the '-speed' command be included?

**Comment by sebastienvercammen** [comment]
In `docs/basic-install/index.rst`, change `cover an instation` to `cover an installation`. Otherwise looking good.

**Comment by HazyArc14** [comment]
There is another pull request currently, "Completing tutorial with a final Pokéstop spin #1857", that adds a Tutorial.md doc. Should I go ahead and add a link to that page for the '-tut' command or wait until after that pull is merged?

**Comment by sebastienvercammen** [comment]
@HazyArc14 You can add it now.

File: docs/basic-install/index.rst [hunk 3]
```diff
@@ -118,12 +118,29 @@ Once those have run, you should be able to start using the application, make sur
...
+   -l "a street address or lat/lng coords here" -k "MAPS_KEY_HERE" \
+   -hk "HASH_KEY_HERE" -cs -ck "CAPTCHA_KEY"
+
+Lets run through this startup command to make sure you understand what configs are being set.
+
+ * -ac accounts.csv
+Load accounts from CSV (Comma Seperated Values) file containing "auth_service,username,passwd" lines. :doc:`Full Info <multi-account>`
+
+ * -tut
+Complete ToS and tutorial steps on accounts if they haven't already. :doc:`Full Info <tutorial>`
```
Comment (sebastienvercammen) [inline docs/basic-install/index.rst]: When I said "go ahead, add the link", I meant an actual `[link](relative/url)`, not a placeholder `<tutorial>`.

Comment (HazyArc14) [inline docs/basic-install/index.rst]: "[Full Info](../extras/tutorial.md)" This? If so I'll change the others to meet that same format.

Comment (sebastienvercammen) [inline docs/basic-install/index.rst]: Yup 👍

File: docs/basic-install/index.rst [hunk 4]
```diff
@@ -16,7 +16,7 @@ Credentials
 ***********
 
  * You'll need an active Pokemon Trainer Club account or Google account
- * Get a :doc:`google-maps`
+ * Get a [Google maps key](./google-maps.md)
```
Comment (sebastienvercammen) [inline docs/basic-install/index.rst]: Local links = `[a](b.md)`.

**Comment by FrostTheFox** [comment]
addressed in e4a7cf67d53e278518e8d946258c7e51a64c360e, just didn't feel like editing the linking on your fork :)

**Pull request closed without merging**